### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by coderschoolreview** [hoangtien15/Homeworksubmission#1]
## Goal of this Assignment
The goal of this assignment was to introduce you to 2 main concepts in Machine Learning: 
- Data Visualization
- Exploratory Data Analysis. 

You learn how to query and clean data using pandas library in Python, make some plots which help to understand more about data with Seaborn library.

## Things you did well:
- You can make use of Pandas functions and indexing operators well. This is a good start !
- This shows that you can get used to doing Python and Pandas if you keep practicing more !

## Things to work on:
- You should improve skills on Python and Pandas. It is fine if you cannot get it done initially, but by practicing you can overcome all errors.
- This homework is fundamental is a good start to deal with further problems in upcoming weeks. So you should get used to manipulating with Pandas as soon as possible.

## To sum up
- If you find Python or Pandas are quite difficult and make you lose behind, feel free to meet me at office hour and i will help you to find it much easier :D.
- Overall, excellent work! Practice makes perfect. Keep improving !
